libsignal-service/src/groups_v2/model.rs

```diff
@@ -30,15 +31,15 @@ impl PartialEq for Member {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub struct PendingMember {
-    pub uuid: Uuid,
+    pub service_id: ServiceId,
```

Comment:
@gferon, what do you think, should we...

```suggestion
    pub service_id: ServiceAddress,
```

instead? I think maybe it's worth dropping `ServiceAddress` altogether, and sub it with `ServiceId` from proto...

Reply:
The only problem is that `ServiceId` might be a bit more annoying to use? (to display for example). We can definitely try.

Reply:
We can postpone that until a future refactoring. I've subbed with serviceaddress now.